posix.mak

```diff
@@ -94,7 +94,7 @@ install: $(TOOLS) $(CURL_TOOLS) $(ROOT)/dustmite
 	cp $^ $(INSTALL_DIR)/bin
 
 clean:
-	rm -f $(ROOT)/dustmite $(TOOLS) $(CURL_TOOLS) $(DOC_TOOLS) $(TAGS) *.o $(ROOT)/*.o
+	rm -rf $(ROOT_OF_THEM_ALL)
```

Comment:
Could you please also rename the `ROOT_OF_THEM_ALL` target to `GENERATED` and `ROOT` to `G`, for consistency with the DMD makefile, and so that we intentionally break any possible scripts that were overriding these variables to `.` or some other directory that should not be deleted?

Reply:
The DMD makefile is actually the exception here, both druntime and phobos makefiles use `ROOT_OF_THEM_ALL`.